device/common/include/traccc/device/fill_prefix_sum.hpp

```diff
@@ -0,0 +1,39 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "traccc/definitions/qualifiers.hpp"
+#include "traccc/device/get_prefix_sum.hpp"
+
+// VecMem include(s).
+#include <vecmem/memory/memory_resource.hpp>
+#include <vecmem/utils/copy.hpp>
+
+namespace traccc::device {
+
+/// Function filling the prefix sum for a "size vector"
+///
+/// This simple function is just meant to translate the received "size vector"
+/// into a "prefix sum". I.e. into a list of index pairs that would allow
+/// visiting all elements of the jagged vector described by this "size vector".
+///
+/// @param[in] globalIndex The index of the current thread
+/// @param[in] sizes_view View on the sizes of the "inner vectors" of a jagged
+/// vector
+/// @param[out] ps_view   View on the result vector of index pairs
+///
+TRACCC_HOST_DEVICE
+void fill_prefix_sum(unsigned int globalIndex,
```

Comment:
Let's use `std::size_t` for the global index, like we do for the other functions.